vaadin-testbench-unit/src/test/java/com/vaadin/flow/component/accordion/AccordionWrapTest.java

```diff
@@ -15,14 +15,11 @@
 
 import com.vaadin.flow.router.RouteConfiguration;
 import com.vaadin.testbench.unit.UIUnitTest;
+import com.vaadin.testbench.unit.ViewPackages;
 
+@ViewPackages(packages = "com.vaadin.flow.component.accordion")
```

Comment:
Could be without a defined package. The test was just written before choosing to use the getClass().getPackageName() which is simpler to write and understand.

Reply:
fixed